app/src/androidTest/java/ch/sdp/vibester/activity/TypingGameActivityTest.kt

```diff
@@ -235,14 +235,15 @@ class TypingGameActivityTest {
 
         val intent =
             Intent(ApplicationProvider.getApplicationContext(), TypingGameActivity::class.java)
+        intent.putExtra("gameManager", gameManager)
         val scn: ActivityScenario<TypingGameActivity> = ActivityScenario.launch(intent)
         val ctx = ApplicationProvider.getApplicationContext() as Context
         createMockDataGetter()
         createMockAuthenticator()
         scn.onActivity { activity ->
             activity.checkAnswer(ctx, songTest, gameManager)
         }
-        assertEquals(true, gameManager.getScore() == 0)
+        assertEquals(true, gameManager.getCorrectSongs().size == 0)

```

Comment:
```suggestion
        assertEquals(0, gameManager.getCorrectSongs().size)
```
Maybe simpler and easier to read